sql/parser/builtins.go

```diff
@@ -549,7 +549,19 @@ var builtins = map[string][]builtin{
 		}),
 	},
 
-	// TODO(thschroeter): implement round
+	"round": {
+		builtin{
+			returnType: DummyFloat,
+			types:      typeList{floatType, intType},
+			fn: func(args DTuple) (Datum, error) {
+				n := args[1].(DInt)
+				// TODO(thschroeter): bounds check on n
```

Comment:
Need to figure out what the bounds for number of digits should be. Also, we need to decide if we want to support rounding in front of the decimal point for negative number of digits (as Postges does).